### DISCUSSION TITLE
Add Paperless-NGX

### DISCUSSION THREAD

**Discussion by ByteFetch** [tteck/Proxmox#141] [Ideas]
Hi,

Would you be interested in creating a script for Paperless-NGX?

This is the newest iteration of the Paperless utility and is being actively maintained now.

https://github.com/paperless-ngx/paperless-ngx

**Comment by strangedesign11** [comment]
@tteck Any chance this could be done? It'd be super helpful as the bare metal installation instructions for paperless aren't great and the docker version of paperless has some issues (namely inotify not working over NFS) that could be solved by running the service in a LXC. I'd be happy to make a donation for your time if its feasible to make a script for this. Thanks for all the work you've done so far.

**Comment by tteck** [comment]
Done!

**Reply by dimatx** [reply]
@tteck Thanks!

Where in your LXC would I need to make changes to the paperless-ngx config to fix CSRF-caused 403 when trying to login (via NginxPM)

![image](https://user-images.githubusercontent.com/24803941/190917786-c31d95e7-00bf-434a-a944-8d37a16e36a0.png)

**Reply by tteck** [reply]
https://github.com/paperless-ngx/paperless-ngx/issues/712

config path `/opt/paperless/paperless.conf`

**Reply by dimmuboy** [reply]
If I can have a question too, can you integrate Tika and Gotenberg please?
https://paperless-ngx.readthedocs.io/en/latest/configuration.html#configuration-tika
